content/docs/start/data-and-model-versioning.md

```diff
@@ -82,35 +84,31 @@ outs:
     path: data.xml
 ```
 
-> \* See
-> [Large Dataset Optimization](/doc/user-guide/large-dataset-optimization) and
-> `dvc config cache` for more info. on file linking.
-
 </details>
 
 ## Storing and sharing
 
 You can upload DVC-tracked data or model files with `dvc push`, so they're
 safely stored [remotely](/doc/command-reference/remote). This also means they
 can be retrieved on other environments later with `dvc pull`. First, we need to
-setup a storage:
+setup a storage location:
```

Comment:
The term "location" works here with as it's somewhat explanatory along with the first mention of "remote storage":
```suggestion
setup a remote storage location:
```